.github/workflows/unit-tests.yml

```diff
@@ -59,6 +59,7 @@ jobs:
       - name: Run unit tests
         env:
           ETH_MAINNET_ENDPOINT: ${{ secrets.ETH_MAINNET_ENDPOINT }}
+          ETH_SEPOLIA_ENDPOINT: ${{ secrets.ETH_SEPOLIA_ENDPOINT }}
```

Comment:
This is all setup if you add a github actions secret for it. The stagenet contract check tests will happen even if it is not set, but they will use `https://rpc.sepolia.org/`, which may or may not be reliable enough.

Reply:
added